static/js/click_handlers.js

```diff
@@ -205,6 +206,11 @@ export function initialize() {
         message_flags.toggle_starred_and_update_server(message);
     });
 
+    $("#main_div").on("click", ".show_readby_users", (e) => {
```

Comment:
I think `show_read_indicators` is probably a better name.

I'd also consider clarifying the interface between this and `read_indicators.js`, to try to do the `rows.js` logic here only.

Reply:
Changed to `show_read_receipts` and moved the `rows.js` logic to this.